src/views/webview-app/store/store.ts

```diff
@@ -5,16 +5,23 @@ import ConnectionModel, {
 } from '../connection-model/connection-model';
 import SSL_METHODS from '../connection-model/constants/ssl-methods';
 import {
+  CONNECTION_STATUS,
   INITIAL_WEBVIEW_VIEW_GLOBAL_VARNAME,
+  MESSAGE_FROM_WEBVIEW_TO_EXTENSION,
   MESSAGE_TYPES,
   WEBVIEW_VIEWS
 } from '../extension-app-message-constants';
 
+interface VSCodeApi {
+  postMessage: (message: MESSAGE_FROM_WEBVIEW_TO_EXTENSION) => void;
+}
+
 // eslint-disable-next-line no-var
-declare var acquireVsCodeApi: any;
+declare var acquireVsCodeApi: () => VSCodeApi;
```

Comment:
Won't window. acquireVsCodeApi work here the same way? To not mix vars with consts and lets.